products/memgraph/upcoming/tutorials/replication.md

```diff
@@ -0,0 +1,135 @@
+## How to Set Up Replication on a Small Cluster (Enterprise)
+
+This article is a part of a series intended to showcase Memgraph's features,
+and bring the user up to speed on developing with Memgraph.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials.md).
+
+### Introduction
+
+This example demonstrates how to create a simple cluster of nodes running
+Memgraph instances, and set up replication. We'll demonstrate the use and
+behavior of the available synchronization modes.
+
+### Cluster topology
+
+The cluster consists of four nodes, a single main and three replicas:
+
+* the main node, containing the original data to be replicated to other nodes,
+* a node that will be replicated to using the sync mode,
+* a node that will be replicated to using the async mode,
+* a node that will be replicated to using the sync-with-timeout mode.
```

Comment:
we need to decide how to call this mode
semi-sync or sync-with-timeout
in the concepts you use semi-sync

Reply:
I vote semi-sync.

Reply:
+1.